UnityProject/Assets/Scripts/Managers/PlayerList.cs

```diff
@@ -44,85 +38,6 @@ private void Awake()
 		}
 	}
 
-	//Called on the server when a kill is confirmed
-	[Server]
-	public void UpdateKillScore(GameObject perpetrator, GameObject victim)
-	{
-		if ( perpetrator == null )
-		{
-			return;
-		}
-
-		var playerName = Get(perpetrator, true).Name;
-		if ( playerScores.ContainsKey(playerName) )
-		{
-			playerScores[playerName]++;
-		}
-
-		JobType perpetratorJob = perpetrator.GetComponent<PlayerScript>().JobType;
-		JobDepartment perpetratorDept = SpawnPoint.GetJobDepartment(perpetratorJob);
-
-		if ( !departmentScores.ContainsKey(perpetratorDept) )
-		{
-			departmentScores.Add(perpetratorDept, 0);
-		}
-
-		if ( victim == null )
-		{
-			return;
-		}
-
-		JobType victimJob = victim.GetComponent<PlayerScript>().JobType;
-		JobDepartment victimDept = SpawnPoint.GetJobDepartment(victimJob);
-
-		if ( perpetratorDept == victimDept )
-		{
-			departmentScores[perpetratorDept]--;
-		}
-		else
-		{
-			departmentScores[perpetratorDept]++;
-		}
-	}
-
-	[Server]
-	public void TryAddScores(string uniqueName)
-	{
-		if ( !playerScores.ContainsKey(uniqueName) )
-		{
-			playerScores.Add(uniqueName, 0);
-		}
-	}
-
-	[Server]
-	public void ReportScores()
-	{
-		/*
-		var scoreSort = playerScores.OrderByDescending(pair => pair.Value)
-		    .ToDictionary(pair => pair.Key, pair => pair.Value);
-
-		foreach (KeyValuePair<string, int> ps in scoreSort)
-		{
-		    UIManager.Chat.ReportToChannel("<b>" + ps.Key + "</b>  total kills:  <b>" + ps.Value + "</b>");
-		}
-		*/
-
-		if ( departmentScores.Count == 0 )
-		{
-			PostToChatMessage.Send("Nobody killed anybody. Fucking hippies.", ChatChannel.System);
-		}
-
-		var scoreSort = departmentScores.OrderByDescending(pair => pair.Value)
-			.ToDictionary(pair => pair.Key, pair => pair.Value);
-
-		foreach ( KeyValuePair<JobDepartment, int> ds in scoreSort )
-		{
-			PostToChatMessage.Send("<b>" + ds.Key + "</b>  total kills:  <b>" + ds.Value + "</b>", ChatChannel.System);
-		}
-
-		PostToChatMessage.Send("Game Restarting in 10 seconds...", ChatChannel.System);
-	}
-
```

Comment:
why was this deleted? Looks like something that could be used in the future

Reply:
Its TDM gamestate.